tools/fuzz/fuzz.cpp

```diff
@@ -734,7 +735,7 @@ int main(int argc, const char** argv) {
     std::string facts_json_string((std::istreambuf_iterator<char>(facts_input)),
                                   std::istreambuf_iterator<char>());
     facts_input.close();
-    if (google::protobuf::util::Status::OK !=
+    if (google::protobuf::util::Status(google::protobuf::util::Status::OK) !=
```

Comment:
I'm confused about this now :)

The reason I had to update the protobuf library was due to this file not compiling with the old protobuf library, and I remember changing these lines to get it to compile and thinking it was weird that I had to do so.

I will investigate further.

Reply:
Trying this locally with the protobuf version used in this PR, the changes are indeed unnecessary, so something is indeed wrong.

Reply:
OK, I have finally figured this out!

It turns out that Chromium apply a patch to protobufs to get rid of static initializers. See:

https://source.chromium.org/chromium/chromium/src/+/main:third_party/protobuf/patches/0003-remove-static-initializers.patch;l=53

As a result, in Chromium, there is StatusPod, which wraps an error code, and Status, which can be created from a StatusPod. `OK` then becomes a static `StatusPod` object (which I guess achieves the objective of avoiding static initializers, since plain old data is handled differently).

So in my change, the seemingly redundant copy constructor call actually turns into a `Status(StatusPod))` constructor call with Chromium.

I think it's best if I write a code comment to briefly explain this, but keep the code as is, since it works both with and without the patch.

Reply:
OK, I have pushed a change that explains the reason for this.